packages/amplication-server/src/core/entity/entity.service.ts

```diff
@@ -846,6 +881,133 @@ export class EntityService {
     });
   }
 
+  async deleteEntityFromSourceResource(
```

Comment:
The name is confusing. Can we change it to:
```suggestion
  async deleteEntityFromSource(
```